namespaces/mson-namespace.md

```diff
@@ -54,6 +54,78 @@ In other words, an expanded element is one that does not contain any _Identifier
 
 The expanded Refract MUST, however, keep the track of what data structure was expanded and what from where.
 
+### Example
+
+Extending the element "A" to form new element "B":
+
+```json
+{
+  "element": "extend",
+  "meta": {
+    "id": "B"
+  },
+  "content": [
+    {
+      "element": "string",
+      "meta": {
+        "id": "A"
+      },
+      "content": "base element content"
+    },
+    {
+      "element": "string",
+      "content": "derived content"
+    }
+  ]
+}
+```
+
+In MSON refract, the following is equivalent:
+
+```json
+{
+  "element": "string",
+  "meta": {
+    "id": "A"
+  },
+  "content": "base element content"
+}
+```
+
+```json
+{
+  "element": "A",
+  "meta": {
+    "id": "B"
+  },
+  "content": "derived content"
+}
+```
+
+Finally, using the `extend` element to expand the MSON refract for id "B":
```

Comment:
This is a little confusing to me. You say "Finally" but I wasn't aware I was being taken through steps on doing something. I think adding a little more clarity here on what you're trying to explain will really help this. I don't think it's currently clear.